webapp/src/routes/app/+page.svelte

```diff
@@ -1,53 +1,221 @@
-                class="map"
-                standardControls
-                style="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
-            />
-        </div>
+<div class="container">
+    <div class="listings" bind:this={listingsContainer}>
+        <Search
+            class="search-input"
+            type="text"
+            bind:value={searchQuery}
+            placeholder="Explore spots around this area..."
+            name="search"
+            on:input={() => {
+                dropdownOpen = true;
+            }}
+        />
+
+        {#if results.length > 0 && dropdownOpen}
+            <ul class="dropdown">
+                {#each results as result}
+                    <!-- svelte-ignore a11y-click-events-have-key-events a11y-no-noninteractive-element-interactions -->
+                    <li on:click={() => handleSelect(result)}>
+                        {result.properties.formatted}
+                    </li>
+                {/each}
+            </ul>
+        {/if}
+
+        {#each spots_data as listing}
+            <div
+                class="booking-info-container {listing.id === selectedListingId ? 'highlight' : ''}"
+                id={`listing-${listing.id}`}
+            >
+                <SpotsListComponent {listing} />
+            </div>
+        {/each}
+
+        {#if showBackToTop}
+            <button class="back-to-top" on:click={scrollToTop}> ↑ Back to Top </button>
+        {/if}
+    </div>
+
+    <div class="map-view">
+        <MapLibre
+            center={mapCenter}
+            {zoom}
+            style="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
+        >
+            <GeolocateControl position="bottom-left" fitBoundsOptions={{ maxZoom: 12 }} />
+            {#each spots_data as { id, location }}
+                <DefaultMarker
+                    lngLat={[location.longitude, location.latitude]}
+                    on:click={() => handleMarkerClick(id)}
+                >
+                    <Popup offset={[0, -10]}>
+                        <div class="text-lg font-bold">
+                            {location.street_address}, {location.city}
+                        </div>
+                        <button on:click={() => handleMarkerClick(id)}>Go to listing</button>
+                    </Popup>
+                </DefaultMarker>
+            {/each}
+        </MapLibre>
     </div>
-</body>
+</div>
 
 <style>
     .container {
         display: flex;
         flex-direction: row;
-        margin: 8.5rem 3rem;
         max-height: fit-content;
     }
 
     .listings {
         display: flex;
-        width: 30%;
-        min-height: 100%;
+        width: 25%;
+        max-height: 82vh;
+        overflow-y: auto;
         flex-direction: column;
-        font-size: 1.2rem;
-        font-weight: bold;
-        align-items: center;
-        justify-content: center;
+        padding: 0.5rem;
         background-color: rgb(186, 214, 183);
```

Comment:
I think we should move away from the green. Doesn't suit our general style anymore lol. how about  #848482